src/map/pc.c

```diff
@@ -1468,12 +1456,9 @@ void pc_reg_received(struct map_session_data *sd)
 		clif_changeoption( &sd->bl );
 	}
 
-	pc_check_expiration(sd);
-
-	if( sd->state.autotrade ) {
-		clif_parse_LoadEndAck(sd->fd, sd);
-		sd->autotrade_tid = add_timer(gettick() + battle_config.feature_autotrade_open_delay, pc_autotrade_timer, sd->bl.id, 0);
-	}
+	intif_storage_request(sd,TABLE_STORAGE); // Request storage data
```

Comment:
Again, I think you merged incorrectly. Check your intif.h to make sure you defined the function in there.